tests/sys_crypto/Makefile

```diff
@@ -2,7 +2,9 @@ include ../Makefile.tests_common
 
 USEMODULE += embunit
 
-USEMODULE += crypto_3des
```

Comment:
This goes into the  3DES removal commit.

Reply:
Done. I hope I separated the commits correctly.